src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1580,6 +1580,9 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     UCT_DC_MLX5_CHECK_FORCE_FULL_HANDSHAKE(self, config, dct, DCT, status, err);
 
     ucs_assert(self->tx.num_dci_pools <= UCT_DC_MLX5_IFACE_MAX_DCI_POOLS);
+    UCS_STATIC_ASSERT((UCT_DC_MLX5_IFACE_MAX_DCI_POOL_INDEX &
```

Comment:
No need to define UCT_DC_MLX5_IFACE_MAX_DCI_POOL_INDEX just for the assert.
we can
UCS_STATIC_ASSERT((UCT_DC_MLX5_IFACE_MAX_DCI_POOLS - 1) <= UCT_DC_MLX5_EP_FLAG_POOL_INDEX_MASK)